bucket.go

```diff
@@ -686,6 +739,10 @@ func (b *Bucket) write() []byte {
 	return value
 }
 
+func (b *Bucket) Write() []byte {
```

Comment:
@ahrtr  i had to add this exposed func as a wrapper to `write()` 

is there another way better than this, now i am changing the `bucket` api